shared/login/register/error/index.js

```diff
@@ -17,6 +17,12 @@ const renderError = (error: RPCError) => {
     return acc
   }, {})
   switch (error.code) {
+    case ConstantsStatusCode.scdeviceprovisionoffline:
+      return (
+        <Text type="Body">
+          Device provisioning failed because this device went offline. Please check your network connection and try again.
```

Comment:
is it this device or the other?

Reply:
@patrickxb, please could you confirm? I originally thought the same as @chrisnojima -- that this was about communicating other-device network status -- but from your repro instructions I think it's actually just same-device network status.

Reply:
Yes, it's "this" device, the device you are currently on.